frontend/src/entry/model/settings/EntityItem.tsx

```diff
@@ -15,13 +15,13 @@ type EntityItemProps = {
   accessList: CollaboratorEntry[]
   onAccessListChange: (value: CollaboratorEntry[]) => void
   model: EntryInterface
+  setRoleError: (value: string) => void
```

Comment:
Two things here. One is that this is an event, so should be prefixed with `on...`. The other is that this should be generic as it is used for two different errors. Consider renaming to `onError` as beyond this we don't care where the error originated, we simply care that there is an error to display to the user.